src/main/java/com/google/sps/user/User.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.user;
+
+import org.jetbrains.annotations.Nullable;
+
+/** Stores the data related to one user. */
+public class User {
+    private final String id, email;
+    private final boolean publicPortfolio;
+    @Nullable 
+    private final String name, selfIntroduction, imgUrl;
+
+    public static class Builder {
```

Comment:
Maybe take a look at AutoValue: https://github.com/google/auto/blob/master/value/userguide/builders.md

It is very nice to use library; removes a lots of boiler plate code.

Reply:
I will consider AutoValue for later classes.